### PR TITLE
Ria 2999 fix repetitive notifications

### PR DESCRIPTION
**Pull request by burale**
### JIRA link (if applicable) ###

[RIA-2999](https://tools.hmcts.net/jira/browse/RIA-2999)

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-allowed-after-respondent-dismissed.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        "subject": "Immigration and Asylum appeal: application to appeal to Upper Tribunal",
+        "body": [
+          "PA/12345/2019",
+          "A1234567",
+          "987654321",
+          "Talha Awan"
+        ]
+      }
+    ],
+    "notifications": [
```
Comment (bradsorour) [inline src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-allowed-after-respondent-dismissed.json:50]: These functional tests will always pass, regardless of the recipient address.

The `"notifications": [` needs to be removed as was done in RIA-1432.

As it stands, with more than one "notifications" node, only one is passed to assert function, which may give you a false positive functional test. These changes will need to made to the rest of the functional tests too.

File: src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-allowed-after-respondent-dismissed.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      {
+        "reference": "1007_FTPA_APPLICATION_DECISION_HOME_OFFICE_APPELLANT",
+        "recipient": "{$respondentEmailAddresses.respondentEvidenceDirection}",
+        "subject": "Immigration and Asylum appeal: application to appeal to Upper Tribunal",
+        "body": [
+          "PA/12345/2019",
+          "A1234567",
+          "987654321",
+          "Talha Awan"
+        ]
```
Comment (bradsorour) [inline src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-allowed-after-respondent-dismissed.json:47]: The following needs to be included in the functional tests (and injected into the GovNotify templates) to follow the same template standard. This was introduced in RIA-2936 (currently in master):

```
"{$iaExUiFrontendUrl}",
"{$customerServices.telephoneNumber}",
"{$customerServices.emailAddress}"
```
The linkToOnlineService also needs to be a dynamic placeholder in the GovNotify templates now (i.e. not hard-coded).

File: src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-granted-after-respondent-granted.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        "subject": "Immigration and Asylum appeal: application to appeal to Upper Tribunal granted",
+        "body": [
+          "PA/12345/2019",
+          "A1234567",
+          "987654321",
+          "Talha Awan"
+        ]
+      }
+    ],
+    "notifications": [
```
Comment (bradsorour) [inline src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-granted-after-respondent-granted.json:50]: Same as above

File: src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-allowed-after-respondent-dismissed.json (hunk 1 above)
Comment (burale) [reply src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-allowed-after-respondent-dismissed.json]: Fixed

File: src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-allowed-after-respondent-dismissed.json (hunk 2 above)
Comment (burale) [reply src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-allowed-after-respondent-dismissed.json:47]: Fixed

**Pull request merged**